Documentation/cluster-crd.md

```diff
@@ -70,6 +70,7 @@ Below are the settings available, both at the cluster and individual node level,
   - `databaseSizeMB`:  The size in MB of a bluestore database.
   - `walSizeMB`:  The size in MB of a bluestore write ahead log (WAL).
   - `journalSizeMB`:  The size in MB of a filestore journal.
+- `directories`: Cluster level list of directories to use for storage. These values will be set for all nodes that have no `directories` set.
```

Comment:
Please indent this properly

Reply:
This is the correct indent. `directories` is not on the level of the `storeConfig`.